### PR TITLE
replaced the prints with logs (rose/client - rose/common)

### PR DESCRIPTION
**Pull request by Abelarm**
Modified the client package, and the common one (common didn't have prints) 

see #190

### CODE REVIEW

File: rose/client/component.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import logging
+
+log = logging.getLogger('Component')
```
Comment (nirs) [inline rose/client/component.py]: If we don't log anything here, we don't need logging yet.

File: rose/client/car.py [hunk 2]
```diff
@@ -1,8 +1,11 @@
+import logging
 import random
 import os
 from rose.common import config
 import component
 
+log = logging.getLogger('car')
```
Comment (nirs) [inline rose/client/car.py]: If we don't log anything here, we don't need logging yet.

File: rose/client/game.py [hunk 3]
```diff
@@ -60,18 +62,18 @@ def car(self):
     # Handling client events
 
     def client_connected(self):
-        print 'client connected: joining as', self.name
+        log.info('client connected: joining as', self.name)
```
Comment (nirs) [inline rose/client/game.py]: When converting prints to logging, we need format string and args - for example:

    log.info('client connected: %s', self.name)

File: rose/client/main.py [hunk 4]
```diff
@@ -25,7 +27,7 @@ def lineReceived(self, line):
         elif msg.action == 'error':
             self.factory.error(msg.payload)
         else:
-            print 'Unexpected message:', msg.action, msg.payload
+            log.info('Unexpected message:', msg.action, msg.payload)
```
Comment (nirs) [inline rose/client/main.py]: Same, need proper format string.

File: rose/client/main.py [hunk 5]
```diff
@@ -67,7 +69,7 @@ def send_message(self, msg):
 
 def main():
```
Comment (nirs) [inline rose/client/main.py]: We need to configure the logger here - see #216

File: rose/client/track.py [hunk 6]
```diff
@@ -1,8 +1,10 @@
 import glob
+import logging
 import os
 from rose.common import config, obstacles
 import component
 
+log = logging.getLogger('main')
```
Comment (nirs) [inline rose/client/track.py]: We don't need logging here yet.

File: rose/client/car.py [hunk 7]
```diff
@@ -1,5 +1,10 @@
+import logging
```
Comment (nirs) [inline rose/client/car.py]: logging is not needed here.

File: rose/client/car.py [hunk 8]
```diff
@@ -1,5 +1,10 @@
...
+import random
+import os
+from rose.common import config
```
Comment (nirs) [inline rose/client/car.py]: These imports were just removed on master and are not needed after #219 and #227.

File: rose/client/car.py [hunk 9]
```diff
@@ -1,5 +1,10 @@
...
 import component
 
+
```
Comment (nirs) [inline rose/client/car.py]: This is correct, but not related to import changes - other people are working on pep8 fixes, and such change will conflict with their changes, see #223.

File: rose/client/game.py [hunk 10]
```diff
@@ -60,18 +62,19 @@ def car(self):
...
+        log.info('client disconnected: %s', reason.getErrorMessage())
 
     def client_failed(self, reason):
-        print 'client failed:', reason.getErrorMessage()
+        log.info('client failed: %s', reason.getErrorMessage())
 
     def client_error(self, error):
-        print 'client error:', error.get('message')
+        log.info('client error: %sp', error.get('message'))
+        log.info('client error: %sp', error.get('message'))
```
Comment (nirs) [inline rose/client/game.py]: Two issues:
- log line duplicated
- trailing "p" at the end of the format string.

File: rose/client/game.py [hunk 11]
```diff
@@ -60,18 +62,18 @@ def car(self):
...
-        print 'client disconnected:', reason.getErrorMessage()
+        log.info('client disconnected: %s', reason.getErrorMessage())
 
     def client_failed(self, reason):
-        print 'client failed:', reason.getErrorMessage()
+        log.info('client failed: %s', reason.getErrorMessage())
 
     def client_error(self, error):
-        print 'client error:', error.get('message')
+        log.info('client error: %sp', error.get('message'))
```
Comment (nirs) [inline rose/client/game.py]: There is still trailing "p" in the format string

Comment (Abelarm) [reply rose/client/game.py]: you're right just amending the commit

Comment (Abelarm) [reply rose/client/game.py]: sorry 👎

**Pull request merged**